dev/view/groups/navigation.html

```diff
@@ -0,0 +1,6 @@
+<ul class="nav nav-pills nav nav-pills secondary-nav">
+    <li class="active"><a href="#tab-public-groups" class="pill-item" data-toggle="pill">Public</a></li>
+    <li><a href="#tab-private-groups" class="pill-item" data-toggle="pill">Private</a></li>
+    <li><a href="#tab-create-groups" class="pill-item" data-toggle="pill">Create</a></li>
+    <li><a href="#tab-manage-groups" class="pill-item" data-toggle="pill">Manage</a></li>
+</ul>
```

Comment:
This file can also be inside `popup_dev.html`.  We basically want our `popup_dev.html` to be like a template containing the structure of our application. We just want to separate the views.